words/objects.txt

```diff
@@ -1919,10 +2128,12 @@ swordtail
 sycamore
 syrup
 system
+syzygy
 table
 tablecloth
 tabletop
 tachometer
+taco
```

Comment:
Sometimes used as a term for vulvas; might be suggestive depending on the predicate used.

Reply:
Maybe quesadilla or torta would be good replacements.

Reply:
mmmm making me hungry ;)